src/odemis/driver/pi.py

```diff
@@ -585,7 +585,7 @@ def convertSmallMToDevice(self, m):
         # dis = a*x² + b*x + c
         # so x: impossible if dis < c => return 0
         #       normal solution (with x >0) : x = (-b + sqrt(b**2 - 4*a*(c-y)))/(2 * a)
-        sign = cmp(m, 0)
+        sign = numpy.sign(m)
```

Comment:
This doesn't do the same thing, does it? 
math.copysign(x,y) -> return x with the sign of y -> `copysign(-1, -1) = -1`
sign(y) * x -> reverse sign if y is negative, otherwise do nothing -> 'sign(-1) * -1 = 1`

Reply:
What might work is `math.copysign(1, m) * duration`. If you don't have any other ideas, I'll go with that one.

Reply:
Indeed, you're completely right: if the second argument is negative, this has a different meaning. Actually, also if x == y, then `cmp()` returns 0.
Anyway, I think that none of these cases can ever happen (I assume that duration is always positive).